dm/worker/subtask.go

```diff
@@ -612,10 +614,24 @@ func (st *SubTask) unitTransWaitCondition(subTaskCtx context.Context) error {
 			if err != nil {
 				return terror.WithClass(err, terror.ClassDMWorker)
 			}
+			if st.cfg.EnableGTID {
```

Comment:
we can put this code block before `pos2, err := utils.DecodeBinlogPosition(relayStatus.RelayBinlog)`.